### PR TITLE
feat: overflow avoidance

### PR DESCRIPTION
**Pull request by 0xClandestine**

### CODE REVIEW

**Comment by 0xClandestine** [comment]
wip

File: src/contracts/libraries/SlashingLib.sol [hunk 1]
```diff
@@ -1,11 +1,15 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity ^0.8.27;
 
+import "@openzeppelin/contracts/utils/math/Math.sol";
```
Comment (gpsanant) [inline src/contracts/libraries/SlashingLib.sol:4]: Nice! pls check out AllocationManager and DelegationManager for more potential overflows

Comment (0xClandestine) [reply src/contracts/libraries/SlashingLib.sol:4]: AM done, DM doesn't need it.

**Comment by gpsanant** [comment]
can we close this? @0xClandestine

**Comment by gpsanant** [comment]
can u make a seperate PR with any AM changes you have?

**Pull request closed without merging**